tests/acceptance/features/apiSharingNgShareInvitation/updateShareInvitations.feature

```diff
@@ -1427,3 +1427,358 @@ Feature: Update permission of a share
       | Space Viewer     |
       | Space Editor     |
       | Manager          |
+
+  @env-config
+  Scenario Outline: update share role of a file to an existing role after assigned share role is disabled (Personal Space)
+    Given the administrator has enabled the permissions role "Secure Viewer"
+    And user "Alice" has uploaded file with content "hello world" to "textfile.txt"
+    And user "Alice" has sent the following resource share invitation:
+      | resource        | textfile.txt  |
+      | space           | Personal      |
+      | sharee          | Brian         |
+      | shareType       | user          |
+      | permissionsRole | Secure Viewer |
+    And the administrator has disabled the permissions role "Secure Viewer"
+    When user "Alice" updates the last resource share with the following properties using the Graph API:
+      | permissionsRole | <new-permissions-role> |
+      | space           | Personal               |
+      | resource        | textfile.txt           |
+    Then the HTTP status code should be "200"
+    And the JSON data of the response should match
+      """
+      {
```

Comment:
the response doesn't mention about the new role? if not then no need to check the response but do check that sharee can/cannot perform actions according to the new role